packages/astro/package.json

```diff
@@ -234,7 +234,7 @@
   "engines": {
     "node": "^18.17.1 || ^20.3.0 || >=21.0.0",
     "npm": ">=9.6.5",
-    "pnpm": ">=7.1.0"
+    "pnpm": ">=9.1.2"
```

Comment:
I don't think we should change this as the `engines` field is for users installing this package, and we say that "you can only download `astro` with pnpm >=7.1.2". We don't want to restrict this to latest.

(Personally I'd remove this line altogether as I'm not sure how useful this way, but probably another topic)

Reply:
Do you know if there's a way with renovate to prevent auto-updating pnpm for this package?